sdk/src/assertions/soft_binding.rs

```diff
@@ -0,0 +1,109 @@
+use serde::{Deserialize, Serialize};
+
+use super::labels;
+use crate::{
+    assertion::{Assertion, AssertionBase, AssertionCbor, AssertionJson},
+    assertions::region_of_interest::RegionOfInterest,
+    cbor_types::UriT,
+    Result,
+};
+
+const ASSERTION_CREATION_VERSION: usize = 1;
```

Comment:
@ok-nick @gpeacock @mauricefisher64 Do we need to use usize for the version numbers? It will work for Rust, but cause us pain if we ever need to have other languages and tools bind to that code (because of the type). It's also a very hypothetical case, but I want to point it out.

Reply:
@tmathern is it usize specifically that is causing the issue or just that the size is typed?

Reply:
The `usize` type specifically (the family of types whose size actually depend on the arch you're running on, as they don't have a generic type that fits all). Not an issue yet, but could become one - so if we could get away with an `int` from the beginning, it would be nice.